pgoapi/pgoapi.py

```diff
@@ -242,11 +259,17 @@ def spin_nearest_fort(self):
         if destinations:
             nearest_fort = destinations[0][0]
             nearest_fort_dis = destinations[0][1]
-            if nearest_fort_dis <= 40.00:
-                self.fort_search_pgoapi(nearest_fort, player_postion=self.get_position(),
-                                        fort_distance=nearest_fort_dis)
-                if 'lure_info' in nearest_fort:
-                    self.disk_encounter_pokemon(nearest_fort['lure_info'])
+            self.log.info('Nearest fort dist %s', nearest_fort_dis)
```

Comment:
I don't think this breaks the feature.  Its just walking closer to the pokestops on the same route to make sure it hits them, then it hops right back on the google route.  The diff didn't update for some reason but I don't go completely out of the way to hit the stops, just if they are within 10 meters